### PR TITLE
Detected cluster details populated only when detected cluster-id is p…

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
…resent

tendrl-bug-id: Tendrl/node-agent#398

Signed-off-by: root <root@dhcp43-207.lab.eng.blr.redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@r0h4n @anmolbabu please review

File: tendrl/node_agent/node_sync/sds_detect.py [hunk 1]
```diff
@@ -32,7 +32,8 @@ def load_and_execute_sds_discovery_plugins():
     # Execute the SDS discovery plugins and tag the nodes with data
     for plugin in sds_discovery_manager.get_available_plugins():
         sds_details = plugin.discover_storage_system()
-        if 'detected_cluster_id' in sds_details:
+        if ('detected_cluster_id' in sds_details) and (
```
Comment (anmolbabu) [inline tendrl/node_agent/node_sync/sds_detect.py]: if (
    'detected_cluster_id' in sds_details and
    sds_details['detected_cluster_id'] != ""
)

according to me looks more readable to me.

Comment (GowthamShanmugam) [inline tendrl/node_agent/node_sync/sds_detect.py]: i will change it

**Comment by GowthamShanmugam** [comment]
@anmolbabu changes done

**Comment by anmolbabu** [comment]
Lgtm

**Pull request merged**